src/modules/75-cf/hooks/useGitSync.tsx

```diff
@@ -276,20 +276,21 @@ export const useGitSync = (): UseGitSync => {
   const saveWithGit = async ({
     featureFlagName,
     featureFlagIdentifier,
-    autoCommitMessage,
+    commitMessage,
     patchInstructions,
     onSave
   }: SaveWithGitArgs): Promise<void> => {
     entityDataRef.current = {
       featureFlagIdentifier,
       featureFlagName,
       instructions: patchInstructions.instructions,
+      commitMessage: commitMessage,
```

Comment:
Super-pickey nitpick
```suggestion
      commitMessage,
```